sources/osg/AutoTransform.js

```diff
@@ -0,0 +1,319 @@
+            this._billboardAttribute.setEnabled( false );
+            stateset.setAttributeAndModes( this._billboardAttribute, StateAttribute.OFF );
+        }
+    },
+
+    getAutoRotateToScreen: function () {
+        return this._autoRotateToScreen;
+    },
+
+    setAutoScaleTransitionWidthRatio: function ( autoScaleTransitionWidthRatio ) {
+        this._autoScaleTransitionWidthRatio = autoScaleTransitionWidthRatio;
+    },
+
+    getAutoScaleTransitionWidthRatio: function () {
+        return this._autoScaleTransitionWidthRatio;
+    },
+
+    // local to "local world" (not Global World)
+    computeLocalToWorldMatrix: function ( matrix /*, nodeVisitor */ ) {
+        if ( this._matrixDirty ) this.computeMatrix();
+        if ( this.referenceFrame === TransformEnums.RELATIVE_RF ) {
+            Matrix.preMult( matrix, this._matrix );
+        } else {
+            Matrix.copy( this._matrix, matrix );
+        }
+    },
+
+    computeMatrix: ( function () {
+        var neg = Vec3.create();
+        return function () {
+            if ( !this._matrixDirty ) return;
+            Matrix.makeRotateFromQuat( this._rotation, this._matrix );
+            //_cachedMatrix.postMultTranslate(_position);
+            Matrix.postMultTranslate( this._matrix, this._position );
+            Matrix.preMultScale( this._matrix, this._scale );
+            Matrix.preMultTranslate( this._matrix, Vec3.neg( this._pivotPoint, neg ) );
+            this._matrixDirty = false;
+        };
+
+    } )(),
+
+    computeWorldToLocalMatrix: ( function () {
+        var neg = Vec3.create();
+        var rotInverse = Quat.create();
+        var scaleInverse = Vec3.create();
+        return function ( matrix /*, nodeVisitor */ ) {
+            if ( this.scale[ 0 ] === 0.0 && this.scale[ 1 ] === 0.0 && this.scale[ 2 ] === 0.0 ) {
+                return false;
+            }
+            scaleInverse[ 0 ] = 1 / this._scale[ 0 ];
+            scaleInverse[ 1 ] = 1 / this._scale[ 1 ];
+            scaleInverse[ 2 ] = 1 / this._scale[ 2 ];
+            if ( this.referenceFrame === TransformEnums.RELATIVE_RF ) {
+                Matrix.postMultTranslate( matrix, Vec3.neg( this._position, neg ) );
+                Matrix.postMultRotate( matrix, Quat.inverse( this._rotation, rotInverse ) );
+                Matrix.postMultScale( matrix, scaleInverse );
+                Matrix.postMultTranslate( matrix, this._pivotPoint );
+            } else { // absolute
+                Matrix.makeRotateFromQuat( Quat.inverse( this._rotation, rotInverse ), this._matrix );
+                Matrix.preMultTranslate( matrix, Vec3.neg( this._position, neg ) );
+                Matrix.postMultScale( matrix, scaleInverse );
+                Matrix.postMultTranslate( matrix, this._pivotPoint );
+            }
+            return true;
+        };
+    } )(),
+
+    computeBound: ( function () {
+        var matrix = Matrix.create();
```

Comment:
It can't work, there is no computeBound function, it's computeBoundingSphere instead.